RxCocoa/iOS/UITabBar+Rx.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  UITabBar+Rx.swift
+//  Rx
+//
+//  Created by Jesse Farless on 5/13/16.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+import Foundation
+import UIKit
+
+#if !RX_NO_MODULE
+import RxSwift
+#endif
+
+/**
+ iOS only
+ */
+#if os(iOS)
+extension UITabBar {
+
+    /**
+     Reactive wrapper for `delegate` message `tabBar:willBeginCustomizingItems:`.
+    */
+    public var rx_willBeginCustomizingItems: ControlEvent<[UITabBarItem]> {
+        let source = rx_delegate.observe(#selector(UITabBarDelegate.tabBar(_:willBeginCustomizingItems:)))
+            .map { a in
+                return try castOrThrow([UITabBarItem].self, a[1])
+            }
+
+        return ControlEvent(events: source)
+    }
+
+    /**
+     Reactive wrapper for `delegate` message `tabBar:didBeginCustomizingItems:`.
+    */
+    public var rx_didBeginCustomizingItems: ControlEvent<[UITabBarItem]> {
+        let source = rx_delegate.observe(#selector(UITabBarDelegate.tabBar(_:didBeginCustomizingItems:)))
+            .map { a in
+                return try castOrThrow([UITabBarItem].self, a[1])
+            }
+
+        return ControlEvent(events: source)
+    }
+
+    /**
+     Reactive wrapper for `delegate` message `tabBar:willEndCustomizingItems:changed:`.
+    */
+    public var rx_willEndCustomizingItems: ControlEvent<([UITabBarItem], Bool)> {
```

Comment:
Hi @solidcell ,

I would like to pull this into `develop`, but there is one thing. Could you make this:

```
public var rx_willEndCustomizing: ControlEvent<(items: [UITabBarItem], changed: Bool)>
```

the same for 
```
public var rx_didEndCustomizing: ControlEvent<(items: [UITabBarItem], changed: Bool)>
```